packages/fast-components-styles-msft/src/utilities/fields.ts

```diff
@@ -0,0 +1,41 @@
+import { applyTypeRampConfig } from "../utilities/typography";
+import { fontWeight } from "../utilities/fonts";
+import { defaultHeight } from "../utilities/height";
+import outlinePattern from "../patterns/outline";
+import typographyPattern from "../patterns/typography";
+import { CSSRules } from "@microsoft/fast-jss-manager";
+import { DesignSystem } from "src/design-system";
+import { density } from "../utilities/density";
+import { applyFocusVisible, toPx } from "@microsoft/fast-jss-utilities";
+import { foregroundNormal } from "../utilities/colors";
+
+export function inputFieldStyle(designSystem: DesignSystem): CSSRules<{}> {
+    return {
+        ...applyTypeRampConfig("t7"),
+        ...outlinePattern.rest,
+        ...typographyPattern.rest,
+        fontFamily: "inherit",
+        fontWeight: fontWeight.normal.toString(),
+        boxSizing: "border-box",
+        borderRadius: toPx(designSystem.cornerRadius),
+        padding: "10px",
+        margin: "0",
+        "&:hover": {
+            ...outlinePattern.hover,
+        },
+        ...applyFocusVisible({
+            ...outlinePattern.focus,
+        }),
+        "&:disabled": {
+            ...outlinePattern.disabled,
+            ...typographyPattern.disabled,
+            cursor: "not-allowed",
+            "&::placeholder": {
+                ...typographyPattern.disabled,
+            },
+        },
+        "&::placeholder": {
+            color: foregroundNormal,
+        },
+    };
+}
```

Comment:
Please break this refactor out into a different PR. Changing text field and text area don't align with this feature's intent.

Reply:
Code climate would have complained if I had repeated the same pattern for a third time. I suppose I could pull this out into a small PR to get in first

Reply:
@chrisdholt pulled out into another [PR](https://github.com/Microsoft/fast-dna/pull/1375)

Reply:
Thanks - Unrelated to code climate, this is just all around different work than the purpose of this PR. I'll take a look over there.